crates/cairo-profiler/resources/versioned_constants_0_13_2_1.json

```diff
@@ -0,0 +1,633 @@
+            "step_gas_cost": 406
+        },
+        "secp256k1_get_point_from_x_gas_cost": {
+            "memory_hole_gas_cost": 20,
+            "range_check_gas_cost": 30,
+            "step_gas_cost": 391
+        },
+        "secp256k1_get_xy_gas_cost": {
+            "memory_hole_gas_cost": 40,
+            "range_check_gas_cost": 11,
+            "step_gas_cost": 239
+        },
+        "secp256k1_mul_gas_cost": {
+            "memory_hole_gas_cost": 2,
+            "range_check_gas_cost": 7045,
+            "step_gas_cost": 76501
+        },
+        "secp256k1_new_gas_cost": {
+            "memory_hole_gas_cost": 40,
+            "range_check_gas_cost": 35,
+            "step_gas_cost": 475
+        },
+        "secp256r1_add_gas_cost": {
+            "range_check_gas_cost": 57,
+            "step_gas_cost": 589
+        },
+        "secp256r1_get_point_from_x_gas_cost": {
+            "memory_hole_gas_cost": 20,
+            "range_check_gas_cost": 44,
+            "step_gas_cost": 510
+        },
+        "secp256r1_get_xy_gas_cost": {
+            "memory_hole_gas_cost": 40,
+            "range_check_gas_cost": 11,
+            "step_gas_cost": 241
+        },
+        "secp256r1_mul_gas_cost": {
+            "memory_hole_gas_cost": 2,
+            "range_check_gas_cost": 13961,
+            "step_gas_cost": 125340
+        },
+        "secp256r1_new_gas_cost": {
+            "memory_hole_gas_cost": 40,
+            "range_check_gas_cost": 49,
+            "step_gas_cost": 594
+        },
+        "send_message_to_l1_gas_cost": {
+            "step_gas_cost": 50,
+            "syscall_base_gas_cost": 1
+        },
+        "sierra_array_len_bound": 4294967296,
+        "step_gas_cost": 100,
+        "storage_read_gas_cost": {
+            "step_gas_cost": 50,
+            "syscall_base_gas_cost": 1
+        },
+        "storage_write_gas_cost": {
+            "step_gas_cost": 50,
+            "syscall_base_gas_cost": 1
+        },
+        "stored_block_hash_buffer": 10,
+        "syscall_base_gas_cost": {
+            "step_gas_cost": 100
+        },
+        "transaction_gas_cost": {
+            "entry_point_gas_cost": 2,
+            "fee_transfer_gas_cost": 1,
+            "step_gas_cost": 100
+        },
+        "transfer_entry_point_selector": "0x83afd3f4caedc6eebf44246fe54e38c95e3179a5ec9ea81740eca5b482d12e",
+        "validate_declare_entry_point_selector": "0x289da278a8dc833409cabfdad1581e8e7d40e42dcaed693fa4008dcdb4963b3",
+        "validate_deploy_entry_point_selector": "0x36fcbf06cd96843058359e1a75928beacfac10727dab22a3972f0af8aa92895",
+        "validate_entry_point_selector": "0x162da33a4585851fe8d3af3c2a9c60b557814e221e0d4f30ff0b2189d9c7775",
+        "validate_rounding_consts": {
+            "validate_block_number_rounding": 100,
+            "validate_timestamp_rounding": 3600
+        },
+        "validated": "VALID"
+    },
+    "os_resources": {
+        "execute_syscalls": {
```

Comment:
We could, but I'd actually be against this for reasons stated here: https://github.com/software-mansion/cairo-profiler/pull/112#discussion_r1812766797

I think this will be more straightforward to just use the file from blockifier and call it a day (especially that we might want to use other things from it in the near future)

Reply:
Sure, as long as passing limited version of the file works it's fine